web-app/app/modules/notification_center_module.rb

```diff
@@ -1,6 +1,11 @@
-module MessagesModule
+module NotificationCenterModule
```

Comment:
Maybe you should remove the word "Module" at the end. It's clear this is a module and making the name `NotificationCenter` will make your code cleaner.

You should also rename this file to `messages.rb` since it's already in the `modules` folder.